python/cudf/cudf/core/dataframe.py

```diff
@@ -371,9 +371,43 @@ def _setitem_tuple_arg(self, key, value):
                 )
             self._frame._data.insert(key[1], new_ser._column)
         else:
-            if is_scalar(value):
-                for col in columns_df._column_names:
-                    self._frame[col].loc[key[0]] = value
+            if is_scalar(value) and not isinstance(
+                self._frame.index, cudf.MultiIndex
```

Comment:
Shouldn't the first condition at the beginning of this function make the `isinstance(self._frame.index, cudf.MultiIndex)` redundant?